webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2024, RTE (https://www.rte-france.com)
+ *
+ * See AUTHORS.txt
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This file is part of the Antares project.
+ */
+
+import { Fragment } from "react";
+import { Box, Typography, Divider, styled } from "@mui/material";
+import { CommandArgsDTO, CommandItem } from "../../commandTypes";
+import {
+  PersonOutlineOutlined,
+  ScheduleOutlined,
+  UpdateOutlined,
+} from "@mui/icons-material";
+import { useTranslation } from "react-i18next";
+import { format, formatDistanceToNow, fromUnixTime } from "date-fns";
+import { enUS, fr } from "date-fns/locale";
+
+const TinyText = styled(Typography)(() => ({
+  fontSize: "12px",
+}));
+
+const StyledDivider = styled(Divider)(() => ({
+  margin: "0 8px",
+  width: "1px",
+}));
+
+const InfoContainer = styled(Box)(() => ({
+  display: "flex",
+  alignItems: "center",
+  gap: "4px",
+}));
+
+interface Props {
+  item: CommandItem;
+}
+
+const formatDate = (timestamp: number) => {
+  const date = fromUnixTime(timestamp);
+  return format(date, "dd/MM/yyyy HH:mm");
+};
+
+const formatDateWithLocale = (timestamp: number, language: string) => {
+  const date = fromUnixTime(timestamp);
+  const locale = language.startsWith("fr") ? fr : enUS;
+
+  return formatDistanceToNow(date, {
+    addSuffix: true,
+    locale,
+  });
+};
+
+function CommandDetails({ item }: Props) {
+  const [t, i18n] = useTranslation();
+
+  const args = item.args as CommandArgsDTO;
+  const argsData = args?.data?.antares || {};
+
+  const details = [
```

Comment:
Use `t("global.unknown")` for all label with `"Unknown"`

Reply:
done